### DISCUSSION TITLE
Implement Wotlk Classic arena point distribution to match increased PvE raid loot

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#6125] [Suggestions]
# tldr

copy pasta from closed ticket. pve progression already has 3 loot tokens drop instead of 1 (this equates to wotlk prepatch in tbc content). adjust arena point reward distribution by the same factor, or implement wotlk classic like point distribution amounts to match Blizzards approach of making things casual friendly (also lowers the impact of low population server)

https://github.com/chromiecraft/chromiecraft/issues/6062

### What client do you play on?

enUS

### Faction

Both

### Content Phase:

Generic

### Current Behaviour

Currently, AZ has 2 options for arena point calculation: before and after season 6 mode from vanilla wotlk.

### Expected Blizzlike Behaviour

With TBC and WotlK Classic, further arena point calculations were introduced, partially re-creating the original state and partially reflecting modern changes. these also shifted over the time and are documented in here https://www.reddit.com/r/classicwow/comments/us1cg5/tbc_arena_calculator_formula_updated_for_s4/
the current formulas are active since season 4 and are the proposal for implementation in here
these calculations overall increase the amount of points gained and shift the focus to the larger brackets (3s and 5s)

they are supposed to be linked with the "personal" rating in wotlk classic and require each player to do 10 games per week - which is different from the current implementations

wotlk classic (past season 4) differences to wotlk vanilla:
- player have a personal rating, instead of team rating and team membership rating. for vanilla calculation, the team rating is taken into account, if the team membership rating is equal to or not less than 150 rating below the teams rating. if the membership rating is 150 or more rating below the team rating, the membership rating is taken into account for the calculation
- the team has to play at least 10 games per week in vanilla. each member has to have atleast 30% of the teams weekly games to be considered for point calculation. in classic, each player has to do 10 games only
- starting with season 6 in vanilla, arena teams were starting at rating 0 instead of 1500. arena points calculated for each week did default to 1500 rating if the players rating was below 1500 (this has been implemented in AZ recently). in classic teams also start at rating 0, but take this already into account. hence, the calculation is not requiring a differentiation if above or below 1500 rating

### Source

formulas:
https://www.desmos.com/calculator/vmehkbt2ur

5s: 1022/ (1+123 * (2.71828^{-.00412x}))+580

penalties factor
2s: 76/100
3s: 88/100

https://tbc.wow-tools.com/tbc_arena_calculator
https://www.curseforge.com/wow/addons/final-arena-calculator

### Steps to reproduce the problem

.

### Extra Notes

the biggest conflict is the personal players rating from classic and the rating structure from vanilla wotlk. for practical purpose, an adoption of the points formula, considering the old systematic of team rating to team membership rating (and the 150 rating below buffer) could be utilized regardless.
